spec/donations_spec.rb

```diff
@@ -4,37 +4,79 @@
 require_relative '../lib/donations'
 
 RSpec.describe Donations do
-  describe 'when successful' do
+  describe 'when donation is successful' do
     before do
       allow(DonationSystem::Payment).to receive(:attempt).and_return([])
     end
 
+    it 'sets the configuration for the Money gem' do
+      expect(I18n.enforce_available_locales).to be_falsy
+    end
+
     it 'sanitizes parameters' do
       allow(Helpers::InputSanitizer).to receive(:execute)
-      described_class.donate('foo' => 'bar')
+      described_class.donate('param' => 'irrelevant')
       expect(Helpers::InputSanitizer).to have_received(:execute)
-        .with('foo' => 'bar')
+        .with('param' => 'irrelevant')
     end
 
     it 'attempts a donation' do
       data = Helpers::DonationData.new(
-        '', '', '', false, '', '', '', '', '', '', '', ''
+        '', '', '', false, '', '', '', '', '', '', '', '', ''
```

Comment:
These structs are getting a little hard to read. What do you think about using Ruby 2.5's new 'keyword_init: true` ?

Reply:
Cool stuff! 👍